experiments/pagination/README.md

```diff
@@ -0,0 +1,97 @@
+# Use of Paging for large sets
+
+This experiment demonstrates why pagination might provide a better experience with more immediate UI feedback and
+less taxing work for the backend to do.  It allows hands-on, immediate feedback on pagination fields.
+
+## Problem
+At times, even after a query has been optimized for efficiency, it can take a long time to complete the search. Such a
+ delay can be a bad experience for the user and uses a lot of system resources. How can that be improved?
+
+## Goals
```

Comment:
From @clotton 
Make the "Goals" section reflect the "Non-Goals" by using a few bullets.